src/UI/templates/default/Button/tpl.primary.html

```diff
@@ -0,0 +1 @@
+<button type="button" class="btn btn-default btn-primary" href="{ACTION}" {DISABLED}>{LABEL}</button>
```

Comment:
Why a separate template? Looks like adding a variable to tpl.standard.html would be ok and reduce effort, if general button HTML needs to be changed.